test/unit/controllers/declaration/NatureOfTransactionControllerSpec.scala

```diff
@@ -111,14 +111,15 @@ class NatureOfTransactionControllerSpec extends ControllerSpec with OptionValues
 
     "return 303 (SEE_OTHER)" when {
 
-      "information provided by user are correct" in {
+      "user provided correct information" in {
 
+        withNewCaching(aDeclaration(withPreviousDocuments(Document("Y", "MCR", "reference", None))))
         val correctForm = Json.toJson(NatureOfTransaction("1"))
 
         val result = controller.saveTransactionType(Mode.Normal)(postRequest(correctForm))
 
         await(result) mustBe aRedirectToTheNextPage
-        thePageNavigatedTo mustBe controllers.declaration.routes.PreviousDocumentsController.displayPage()
+        thePageNavigatedTo mustBe controllers.declaration.routes.PreviousDocumentsSummaryController.displayPage()
```

Comment:
Should this redirect to Summary, or the page to add first Document?

Reply:
To the summary, Unit tests didn't check if the summary controller logic will redirect somewhere else